scripts/cibuild

```diff
@@ -29,6 +29,7 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
 
         ./scripts/bootstrap --env
         ./scripts/bootstrap --sbtopts
+        ./scripts/load_development_data --download
```

Comment:
A development-_ish_ database needs to be used so that the right users are there for tests